src/main/java/org/broadinstitute/hellbender/engine/dataflow/transforms/examples/transformExample.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.hellbender.engine.dataflow.transforms.examples;
+
+import com.google.cloud.dataflow.sdk.transforms.DoFn;
+import com.google.cloud.dataflow.sdk.transforms.PTransform;
+import com.google.cloud.dataflow.sdk.transforms.ParDo;
+import com.google.cloud.dataflow.sdk.values.KV;
+import com.google.cloud.dataflow.sdk.values.PCollection;
+import org.broadinstitute.hellbender.engine.dataflow.datasources.ReferenceShard;
+
+/**
+ * transformExample is a simple transform that takes Integers and produces a KV pair of ReferenceShard and the same
+ * Integer. The purpose is to show the minimum pieces required to make a PTransform.
+ */
+public class transformExample extends PTransform<PCollection<Integer>, PCollection<KV<ReferenceShard, Integer>>> {
```

Comment:
If you want to keep this, it should be uppercased and, to follow Broad conventions, `final`.